api/src/main/java/org/openmrs/module/oauth2login/authscheme/OAuth2UserInfoAuthenticationScheme.java

```diff
@@ -174,9 +172,7 @@ private void deactivateProviderAccount(User user) {
 			Context.addProxyPrivilege(PrivilegeConstants.MANAGE_PROVIDERS);
 			
 			Collection<Provider> possibleProvider = ps.getProvidersByPerson(user.getPerson());
-			for (Provider provider : possibleProvider) {
-				ps.retireProvider(provider, "Disabling provider account by oaut2login");
-			}
+			possibleProvider.stream().forEach(provider -> ps.retireProvider(provider, "Disabling provider account by oaut2login"));
```

Comment:
Use the constant defined for `oaut2login`: https://github.com/openmrs/openmrs-module-oauth2login/blob/79e3af284f2d8ef7fb29b1a45258a1dbb910e49a/api/src/main/java/org/openmrs/module/oauth2login/OAuth2LoginConstants.java#L19